app/Http/Controllers/StudentsCouncil/AnonymousQuestionController.php

```diff
@@ -37,36 +35,34 @@ public function __construct()
      * the list of questions
      * and the option to add new ones.
      */
-    public function indexSemesters()
+    public function index()
     {
         $this->authorize('administer', AnswerSheet::class);
 
-        return view('student-council.anonymous-questions.index_semesters');
+        return view('student-council.anonymous-questions.index');
     }
 
     /**
      * Returns the 'new question' page.
      */
-    public function create(Semester $semester)
+    public function create()
     {
         $this->authorize('administer', AnswerSheet::class);
 
-        if ($semester->isClosed()) {
+        if (!$this->isActive()) {
             abort(403, "tried to add a question to a closed semester");
         }
-        return view('student-council.anonymous-questions.create', [
-            "semester" => $semester
-        ]);
+        return view('student-council.anonymous-questions.create');
```

Comment:
This here should remain in the backend, I think. We could choose it in the Blade file, too, but it would make that less general.